hacking/README.md

```diff
@@ -17,7 +17,7 @@ and do not wish to install them from your operating system package manager, you
 can install them from pip
 
     $ easy_install pip               # if pip is not already available
-    $ pip install pyyaml jinja2 nose pytest passlib pycrypto
+    $ pip install pyyaml jinja2 nose pytest passlib cryptography
```

Comment:
should change this to use the requirements.txt file from the source checkout.  Can further advise to pip install cryptography (as the current default is pycrypto)